sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/AvroUtils.java

```diff
@@ -75,8 +75,8 @@
   static {
     // This works around a bug in the Avro library (AVRO-1891) around SpecificRecord's handling
     // of DateTime types.
-    SpecificData.get().addLogicalTypeConversion(new TimeConversions.TimestampConversion());
-    GenericData.get().addLogicalTypeConversion(new TimeConversions.TimestampConversion());
+    SpecificData.get().addLogicalTypeConversion(new JodaTimeConversions.TimestampConversion());
+    GenericData.get().addLogicalTypeConversion(new JodaTimeConversions.TimestampConversion());
```

Comment:
This is necessary because `TimeConversions` converts to `java.time` now?

Maybe we need to consider adding support for `java.time` in `AvroSchemaProvider` now

Reply:
It was renamed to JodatimeConversions in https://github.com/apache/avro/pull/248 for "clarity".

It says it defaults to joda time, but that doesn't explain why it's forcing me to update the tests since I haven't changed this flag value.

Reply:
Hello!  Just for information, Avro 1.9.x supports `joda-time`  and `java.time`, with the strong recommendation that users start preferring the `java.time` classes.  Avro 1.10.x removes built-in support for joda. 

In any case, good news: AVRO-1891 has been fixed in Avro 1.9.x, so these lines can probably be removed!